container/root/etc/cont-init.d/09-php-fpm.sh

```diff
@@ -1,25 +0,0 @@
-#!/usr/bin/with-contenv bash
```

Comment:
all vars in here converted to full environment var representation, runtime config `sed`'ing no longer required